src/apollo/queries.js

```diff
@@ -18,34 +18,6 @@ export const SUBGRAPH_HEALTH = gql`
   }
 `
 
-export const V1_DATA_QUERY = gql`
```

Comment:
is V1_DATA_QUERY used in other places? I'm assuming not since V1 is largely deprecated. If this is the case we can remove it and the this PR can note its removal for posterity